content/en/blog/2023/secure-apps-with-istio/index.md

```diff
@@ -0,0 +1,355 @@
+
+The encrypted data sent over the network from source to destination can’t be modified by any
+identities other than the source and destination once it is sent. In other words, data received is
+the same as data sent. If you don’t have [data integrity](https://en.wikipedia.org/wiki/Data_integrity),
+someone in the middle could modify some bits or the entire content of the data during the
+communication between the source and destination.
+
+### Access Policy Enforcement
+
+Application owners need to apply access policies to their applications and have them enforced
+properly, consistently, and unambiguously. In order to apply policy for both ends of a communication
+channel, we need an application identity for each end. Once we have a cryptographically verifiable
+identity with an unambiguous provenance chain for both ends of a potential communication channel, we
+can begin to apply policies about who can communicate with what. Standard TLS, the widely used
+cryptographic protocol that secures communication between clients (e.g., web browsers) and servers
+(e.g., web servers), only really verifies and mandates an identity for one side - the server. But
+for comprehensive end-to-end policy enforcement, it is critical to have a reliable, verifiable,
+unambiguous identity for both sides - client and server. This is a common requirement for internal
+applications - imagine for example a scenario where only a `frontend` application should call the
+**GET** method for a backend `checkout` application, but should not be allowed to call the POST or
+DELETE method. Or a scenario where only applications that have a JWT token issued by a particular
+JWT issuer can call the GET method for a `checkout` application. By leveraging cryptographic
+identities on both ends, we can ensure powerful access policies are enforced correctly, securely,
+and reliably, with a validatable audit trail.
+
+### FIPS compliance
+
+[Federal Information Processing Standards (FIPS)](https://www.nist.gov/standardsgov/compliance-faqs-federal-information-processing-standards-fips)
+are standards and guidelines for federal computer systems that are developed by
+[National Institute of Standards and Technology (NIST)](https://www.nist.gov/). Not everyone
+requires FIPS compliance, but FIPS compliance means meeting all the necessary security requirements
+established by the U.S. government for protecting sensitive information. It is required when working
+with the federal government. To follow the guidelines developed by the US government relating to
+cybersecurity, many in the private sector voluntarily use these FIPS standards.
+
+To illustrate the above secure application requirements (identity, confidentiality and integrity),
+let’s use two simple applications (an application called `frontend` that calls an application called
+`checkout`). Remember, you can think of “ID” in the diagram as any kind of identity document such as
+a government issued passport, photo identifier:
```

Comment:
`frontend` and `backend` were already mentioned in the Access Policy Enforcement section above without an introduction. Consider moving the introduction to these application earlier in the blog.

Reply:
Updated, removed the intro, feels not needed